screenpy/actions/make_note.py

```diff
@@ -57,12 +57,16 @@ def perform_as(self, the_actor: Actor) -> None:
         if self.key is None:
             raise UnableToAct("No key was provided to name this note.")
 
-        if hasattr(self.question, "answered_by"):
-            value = self.question.answered_by(the_actor)
+        if isinstance(self.question, Answerable):
+            value: object = self.question.answered_by(the_actor)
         else:
             # must be a value instead of a question!
             value = self.question
 
+        if isinstance(self.question, ErrorKeeper):
+            aside(f"Making note of {self.question}...")
+            aside(f"Caught Exception: {self.question.caught_exception}")
```

Comment:
What does this look like in the logs? I'm particularly curious about the "Making note of..." part.

Reply:
```
11:36:01   INFO Tester jots something down under "test".
11:36:01   INFO     Making note of <Mock id='4553053312'>...
11:36:01   INFO     Caught Exception: <Mock name='mock.caught_exception' id='4553054368'>
```